File: Basic-Data-Structures-Js-Py/Basic-Data-Structures.py
# === 1 ===
yourArray = [1, "apple", True, 4, 5]

# === 2 === 
myArray = ["a", "b", "c", "d"]
myArray[1] = "banana"

# === 3 ===
def mixedNumbers(lst):
    lst[0:0] = ['I', 2, 'three']
    lst[len(lst):] = [7, 'VIII', 9]
    # or... lst.extend([7, 'VIII', 9])
    return lst
# print(mixedNumbers(['IV', 5, 'six']))

# === 4 ===
def popPop(lst):
    popped = lst.pop() # removes last element 
    shifted = lst.pop(0) # removes 0th element
    return [shifted, popped]
# print(popPop(['challenge', 'is', 'not', 'complete']))

File: Basic-Data-Structures-Js-Py/Basic-Data-Structures.js
// === 1 Use and Array to Store a Collection of Data ===
let yourArray = [1, "apple", true, 4, 5]

// === 2 Access an Array's Contents Using Bracket Notation ===
let myArray = ["a", "b", "c", "d"]
myArray[1] = "banana"

// === 3 Add Items to an Array ===
function mixedNumbers(arr) {
    arr.unshift('I', 2, 'three')
    arr.push(7, 'VIII', 9)
    return arr
}
// console.log(mixedNumbers(['IV', 5, 'six']))

// === 4 Remove Items from an Array with pop() and shift() ===
function popShift(arr) {
    let popped = arr.pop()
    let shifted = arr.shift()
    return [shifted, popped]
}
// console.log(popShift(['challenge', 'is', 'not', 'complete']))

// === 5 Remove Items Using splice() ===